### PR TITLE
ref(security-dialog) Use new input component for password

### PR DESCRIPTION
**Pull request by robertpin**

### CODE REVIEW

File: react/features/base/i18n/functions.tsx [hunk 1]
```diff
@@ -24,7 +24,7 @@ export async function changeLanguageBundle(language: string, url: string) {
  * @returns {Component} The React Component which wraps {@link component} and
  * enables translations in it.
  */
-export function translate(component: any) {
+export function translate<P extends WithTranslation>(component: React.ComponentType<P>) {
```
Comment (saghul) [inline react/features/base/i18n/functions.tsx:27]: This change doesn't belong in here, can you split it out please?

**Review by saghul — COMMENTED** [review]
Looks like some tests might need adjusting.

File: react/features/base/i18n/functions.tsx (hunk 1 above)
Comment (robertpin) [reply react/features/base/i18n/functions.tsx:27]: I don't understand

**Comment by robertpin** [comment]
Torture PR https://github.com/jitsi/jitsi-meet-torture/pull/523

**Review by saghul — APPROVED** [review]

**Pull request merged**